ucf-vertical-slide.php

```diff
@@ -11,3 +11,68 @@
 if ( ! defined( 'WPINC' ) ) {
     die;
 }
+
+define( 'UCF_VERTICAL_SLIDE__PLUGIN_URL', plugins_url( basename( dirname( __FILE__ ) ) ) );
+define( 'UCF_VERTICAL_SLIDE__STATIC_URL', UCF_VERTICAL_SLIDE__PLUGIN_URL . '/static' );
+define( 'UCF_VERTICAL_SLIDE__PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
+define( 'UCF_VERTICAL_SLIDE__FILEPATH', __FILE__ );
+
+// adding the custom post type path
+include_once 'includes/ucf-vertical-slider-posttype.php';
+
+// Shortcode function
+
+if ( ! function_exists( 'ucf_vertical_slide_shortcode' ) ) {
+
+	function ucf_vertical_slide_shortcode( $atts ) {
+		ob_start();
+		$defaults = array(
+			'slug' => 'default_slug',
+			'id' => 'default_id'
+		);
+
+		// Getting the Id and Slug from shortcode
+		$atts = shortcode_atts($defaults, $atts, 'ucf_vertical_slide');
+		$slider_slug = $atts['slug'];
+		$slider_id = $atts['id'];
```

Comment:
You will need to handle both of these parameters being passed in. So, if a slug is passed in, but not an ID, you need to go find the post using the slug and _get_ the id. If the id is passed in but not the slug, you're all good to go. If both are passed in, you probably want to ignore the slug and use the id.

Reply:
I will handle this in different PR.